### PR TITLE
Updating Documentation with more examples

### LINKED ISSUE #7: Enhancement request - more examples please
Firstly, thanks for this great project. After installing with helm, I’ve successfully managed to sync my first secret using the `app-secret-key` example from the readme.  Where I’ve struggled is creating the second example `app-database-auth`, as despite looking at the code and unit tests I can’t mange to wrangle the correct yaml dict format into the Bitwarden secret.

Would it be possible to add examples into the readme for how to do this, possibly covering similar grounds to the unit tests?

### PR DESCRIPTION
**Pull request by 9strands**
Updating with usage examples and examples of how to create entries in the BitWarden(TM) Secrets Manager product

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -84,12 +84,41 @@ spec:
       password:
         secret: app_database_auth
         key: password
+      database:
+        secret: app_database_auth
+        key: dbname
+      port:
+        secret: app_database_auth
+        key: dbport
```
Comment (jkupferer) [inline README.adoc:92]: I'm not sure if adding more values to this secret helps explain anything? If we do add anything then we should also make sure the comment above is updated? but remember it is just an example to show how the secrets manager works, it doesn't have to include everything.

File: README.adoc [hunk 2]
```diff
@@ -84,12 +84,41 @@ spec:
...
+app_database_auth
+-------------------------------------------------------------------------------
+
+Value:
+--------------------------------------------------------------------------------
+server: productiondb.example.com
+password: <our_really_strong_c0mpl#X_PASSWORD!>
+username: tannerite
+dbname: tracking
+dbport: 12345
```
Comment (jkupferer) [inline README.adoc:118]: I would consider not adding dbname and dbport just to keep the example shorter.

Comment (9strands) [reply README.adoc:118]: I was intending to show that we can store the secret with a value in Secrets Manager and store it with a different entry name in OCP.

The first three match entirely, but dbname and dbport are stored with a different name in k8s than it's stored in the Secret Value.

File: README.adoc (hunk 1 above)
Comment (9strands) [reply README.adoc:92]: I tried to show that we can store the secret with a name that is different than the value it's listed in the yaml in the secret itself.

**Review by jkupferer — APPROVED** [review]

**Pull request merged**